ap_src/ap_app/utils/switch_permissions.py

```diff
@@ -0,0 +1,68 @@
+# Models
+from ..models import (UserProfile)
+from django.contrib.auth.models import User
+
+# Utils
```

Comment:
You don't need this comment.

Reply:
The comments were simply for my benefit during refactoring but I suppose they are no longer needed.

Reply:
Fixed with 717b45d